libcxx/include/__exception/exception_ptr.h

```diff
@@ -26,6 +31,21 @@ namespace std { // purposefully not using versioning namespace
 class _LIBCPP_EXPORTED_FROM_ABI exception_ptr {
   void* __ptr_;
 
+#  if _LIBCPP_AVAILABILITY_HAS_INIT_PRIMARY_EXCEPTION
```

Comment:
This shouldn't be guarded by `_LIBCPP_AVAILABILITY_HAS_INIT_PRIMARY_EXCEPTION`, only the part where you *use* `__init_native_exception` should be guarded.

Reply:
Done